client/lib/desktop/index.js

```diff
@@ -14,22 +13,23 @@ import { newPost } from 'lib/paths';
 import userFactory from 'lib/user';
 const user = userFactory();
 import { ipcRenderer as ipc } from 'electron'; // From Electron
-import store from 'store';
 import * as oAuthToken from 'lib/oauth-token';
 import userUtilities from 'lib/user/utils';
-import location from 'lib/route/page-notifier';
 import { getStatsPathForTab } from 'lib/route';
+import { getReduxStore, reduxGetState } from 'lib/redux-bridge';
+import hasUnseenNotifications from 'state/selectors/has-unseen-notifications';
+import isNotificationsOpen from 'state/selectors/is-notifications-open';
+import { toggleNotificationsPanel, navigate } from 'state/ui/actions';
 
 /**
  * Module variables
  */
-const widgetDomain = 'https://widgets.wp.com';
 
 const Desktop = {
 	/**
 	 * Bootstraps network connection status change handler.
 	 */
-	init: function() {
+	init: async function() {
```

Comment:
I presume that since this is desktop code that we can use `async` / `await`?

cc: @blowery

Reply:
We can (and do) use async/await everywhere in Calypso. Technically, all the Babel transforms are in place. And "officially", we approved the usage on a framework hangout few months ago.